config/stored_requests_test.go

```diff
@@ -154,6 +157,100 @@ func TestInMemoryCacheValidationSingleCache(t *testing.T) {
 	}).validate(AccountDataType, nil))
 }
 
+func TestPostgresConfigValidation(t *testing.T) {
+	tests := []struct {
+		description            string
+		connectionStr          string
+		cacheInitQuery         string
+		cacheInitTimeout       int
+		cacheUpdateQuery       string
+		cacheUpdateRefreshRate int
+		cacheUpdateTimeout     int
+		existingErrors         []error
+		wantErrorCount         int
+	}{
+		{
+			description:   "No connection string",
+			connectionStr: "",
+		},
+		{
+			description:   "Connection string but no queries",
+			connectionStr: "some-connection-string",
+		},
+		{
+			description:    "Valid cache init query with zero timeout",
```

Comment:
Consider defining the 0 timeout since it's mentioned in the test description. I know it's defaulted to 0, but I think it would help with readability.